Modules/Microsoft365DSC/Modules/M365DSCPermissions.psm1

```diff
@@ -1354,28 +1358,21 @@ function Update-M365DSCAzureAdApplication
         return
     }
 
-    $context = Get-AzContext
-    if ($null -eq $context)
-    {
-        Connect-AzAccount
-        $context = Get-AzContext
-    }
-
```

Comment:
Shouldn't we include some authentication to the Graph befor calling the Get-MgServicePrincipal cmdlet?